profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -151,8 +153,40 @@ function _nexteuropa_varnish_get_paths_to_purge($content_type) {
       'nexteuropa_varnish_cache_purge_rule',
       array_keys($result['nexteuropa_varnish_cache_purge_rule'])
     );
+    /** @var \Drupal\nexteuropa_varnish\Entity\PurgeRule $purge_rule */
```

Comment:
I could entirely remove the line, which will actually be worse towards developers, espcially when you are using a proper IDE.

What is most important? Following the exact rules like they are written down like a blind follower - our code sniffer even did not fail on it AFAIK - or improve code readability and automatic introspection by IDEs?